### PR TITLE
update tips for the release of Node.js

### PR DESCRIPTION
**Pull request by ghost**
Refs: https://github.com/nodejs/nodejs.org/pull/4865.

### CODE REVIEW

**Comment by ghost** [comment]
Ping @nodejs/translators：After mine are submitted, you can update to your own languages.

**Comment by Trott** [comment]
If people could put their translations in as [GitHub code suggestions](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request), that would keep it all in this one PR which I think would be useful.

File: locale/fr/index.md [hunk 1]
```diff
@@ -11,8 +11,8 @@ labels:
   tagline-lts: Recommandé pour la plupart des utilisateurs
   changelog: Journal des modifications
   api: Documentation API
-  version-schedule-prompt: Ou regardez le
-  version-schedule-prompt-link-text: Planning LTS
+  version-schedule-prompt: For information about supported releases, see the
+  version-schedule-prompt-link-text: release schedule
```
Comment (aduh95) [inline locale/fr/index.md]:
```suggestion
  version-schedule-prompt: Pour plus d'informations concernant les versions supportées, regardez le
  version-schedule-prompt-link-text: planning des futures versions
```

File: locale/nl/index.md [hunk 2]
```diff
@@ -11,8 +11,8 @@ labels:
   tagline-lts: Aanbevolen Voor Meeste Gebruikers
   changelog: Changelog
   api: API Documentatie
-  version-schedule-prompt: Of bekijk de
-  version-schedule-prompt-link-text: Long Term Support (LTS) schedule
+  version-schedule-prompt: For information about supported releases, see the
+  version-schedule-prompt-link-text: release schedule
```
Comment (hursittarcan) [inline locale/nl/index.md]:
```suggestion
  version-schedule-prompt: Voor meer informatie over ondersteunde releases, zie
  version-schedule-prompt-link-text: releaseschema
```

File: locale/tr/index.md [hunk 3]
```diff
@@ -11,8 +11,8 @@ labels:
   tagline-lts: Çoğu kullanıcı için önerilir
   changelog: Değişiklikler
   api: API Belgeleri
-  version-schedule-prompt: Ya da
-  version-schedule-prompt-link-text: Uzun Süreli Destek (LTS) takvimine bakın
+  version-schedule-prompt: For information about supported releases, see the
+  version-schedule-prompt-link-text: release schedule
```
Comment (hursittarcan) [inline locale/tr/index.md]:
```suggestion
  version-schedule-prompt: Desteklenen sürümler hakkında daha fazla bilgi için
  version-schedule-prompt-link-text: sürüm planına bakıniz
```

File: locale/de/index.md [hunk 4]
```diff
@@ -11,8 +11,8 @@ labels:
   tagline-lts: Für die meisten Nutzer empfohlen
   changelog: Änderungshistorie
   api: API Doku
-  version-schedule-prompt: Oder wirf einen Blick auf den
-  version-schedule-prompt-link-text: LTS Release Plan
+  version-schedule-prompt: For information about supported releases, see the
+  version-schedule-prompt-link-text: release schedule
```
Comment (hursittarcan) [inline locale/de/index.md]:
```suggestion
  version-schedule-prompt: Weitere informationen zu unterstützten versionen finden Sie im
  version-schedule-prompt-link-text: releaseplan
```

File: locale/nl/index.md (hunk 2 above)
Comment (hursittarcan) [reply locale/nl/index.md]: @nodejs/nodejs-nl

File: locale/tr/index.md (hunk 3 above)
Comment (hursittarcan) [reply locale/tr/index.md]: @nodejs/nodejs-tr

File: locale/de/index.md (hunk 4 above)
Comment (hursittarcan) [reply locale/de/index.md]: @nodejs/nodejs-de

File: locale/es/index.md [hunk 5]
```diff
@@ -11,8 +11,8 @@ labels:
   tagline-lts: Recomendado para la mayoría
   changelog: Cambios
   api: Documentación de la API
-  version-schedule-prompt: O eche un vistazo al
-  version-schedule-prompt-link-text: Programa de soporte a largo plazo (LTS)
+  version-schedule-prompt: For information about supported releases, see the
```
Comment (vrunoa) [inline locale/es/index.md]: Para información acerca de los releases soportados, mira

cc @nodejs/nodejs-es

File: locale/es/index.md [hunk 6]
```diff
@@ -11,8 +11,8 @@ labels:
...
+  version-schedule-prompt-link-text: release schedule
```
Comment (vrunoa) [inline locale/es/index.md]: el calendario de lanzamientos

cc @nodejs/nodejs-es

File: locale/es/index.md (hunk 5 above)
Comment (Trott) [reply locale/es/index.md]:
```suggestion
  version-schedule-prompt: Para información acerca de los releases soportados, mira
```

File: locale/es/index.md (hunk 6 above)
Comment (Trott) [reply locale/es/index.md]:
```suggestion
  version-schedule-prompt-link-text: el calendario de lanzamientos
```

File: locale/ka/index.md [hunk 7]
```diff
@@ -11,8 +11,8 @@ labels:
   tagline-lts: რეკომენდებულ­ია მომხმარებელთა უმეტესობისათვის
   changelog: ცვლილებათა ჩამონათვალი
   api: API-დოკუმენტაცია
-  version-schedule-prompt: ან გადახედეთ
-  version-schedule-prompt-link-text: გრძელვადიანი მხარდაჭერის (LTS) განრიგს
+  version-schedule-prompt: For information about supported releases, see the
+  version-schedule-prompt-link-text: release schedule
```
Comment (davidkadaria) [inline locale/ka/index.md]:
```suggestion
  version-schedule-prompt: მხარდაჭერილი გამოშვებების შესახებ ინფორმაციის მისაღებად, იხილეთ
  version-schedule-prompt-link-text: გამოშვების განრიგი
```

**Comment by ghost** [comment]
@nodejs/nodejs-cn ：Please have a review the Chinese version first.

File: locale/zh-cn/index.md [hunk 8]
```diff
@@ -11,8 +11,8 @@ labels:
   tagline-lts: 推荐多数用户使用
   changelog: 更新日志
   api: API 文档
-  version-schedule-prompt: 可参考
-  version-schedule-prompt-link-text: 长期维护版（LTS）日程
+  version-schedule-prompt: 欲知受支持版本的信息，请参阅
+  version-schedule-prompt-link-text: 发布计划
```
Comment (fengmk2) [inline locale/zh-cn/index.md:15]: LGTM

File: locale/tr/index.md [hunk 9]
```diff
@@ -11,8 +11,8 @@ labels:
   tagline-lts: Çoğu kullanıcı için önerilir
   changelog: Değişiklikler
   api: API Belgeleri
-  version-schedule-prompt: Ya da
-  version-schedule-prompt-link-text: Uzun Süreli Destek (LTS) takvimine bakın
+  version-schedule-prompt: Desteklenen sürümler hakkında daha fazla bilgi için
+  version-schedule-prompt-link-text: sürüm planına bakıniz
```
Comment (anonrig) [inline locale/tr/index.md]: The exact translation for this line is “Look at the version release schedule”  If that’s the case this should be “sürüm yayınlanma planına bakınız". If the translation is for “Release schedule” only: it should be “Sürüm yayınlanma planı"

**Review by anonrig — CHANGES_REQUESTED** [review]

File: locale/tr/index.md (hunk 9 above)
Comment (Trott) [reply locale/tr/index.md]:
```suggestion
  version-schedule-prompt-link-text: sürüm yayınlanma planına bakınız
```

File: locale/nl/index.md (hunk 2 above)
Comment (AdriVanHoudt) [reply locale/nl/index.md]: LGTM

File: locale/de/index.md [hunk 10]
```diff
@@ -11,8 +11,8 @@ labels:
   tagline-lts: Für die meisten Nutzer empfohlen
   changelog: Änderungshistorie
   api: API Doku
-  version-schedule-prompt: Oder wirf einen Blick auf den
-  version-schedule-prompt-link-text: LTS Release Plan
+  version-schedule-prompt: Weitere informationen zu unterstützten versionen finden Sie im
+  version-schedule-prompt-link-text: releaseplan
```
Comment (tniessen) [inline locale/de/index.md]:
```suggestion
  version-schedule-prompt: Weitere Informationen zu unterstützten Versionen finden Sie im
  version-schedule-prompt-link-text: Release-Terminplan
```

File: locale/ar/index.md [hunk 11]
```diff
@@ -11,8 +11,8 @@ labels:
   tagline-lts: موصى به لجميع المستخدمين
   changelog: سجل التغييرات
   api: API Docs
-  version-schedule-prompt: أو انظر إلى
-  version-schedule-prompt-link-text: أجندة LTS
+  version-schedule-prompt: For information about supported releases, see the
+  version-schedule-prompt-link-text: release schedule
```
Comment (Trott) [inline locale/ar/index.md:15]: @nodejs-ar Can someone provide translations for these phrases (which will be put together to form a sentence in the page)?

Comment (Trott) [inline locale/ar/index.md:15]: @nodejs/nodejs-ar Can someone provide translations for these phrases (which will be put together to form a sentence in the page)?

File: locale/fa/index.md [hunk 12]
```diff
@@ -11,8 +11,8 @@ labels:
   tagline-lts: پیشنهاد شده برای بیشتر کاربران
   changelog: گزارش تغییرات
   api: مستندات API
-  version-schedule-prompt: یا نگاهی بیاندازید به
-  version-schedule-prompt-link-text: زمان بندی پشتیبانی طولانی مدت
+  version-schedule-prompt: For information about supported releases, see the
+  version-schedule-prompt-link-text: release schedule
```
Comment (Trott) [inline locale/fa/index.md:15]: @nodejs/nodejs-fa  Can someone provide translations for these phrases (which will be put together to form a sentence in the page)?

File: locale/it/index.md [hunk 13]
```diff
@@ -11,8 +11,8 @@ labels:
   tagline-lts: Consigliata
   changelog: Changelog
   api: Documentazione API
-  version-schedule-prompt: Dai un'occhiata alla
-  version-schedule-prompt-link-text: tabella di marcia LTS
+  version-schedule-prompt: For information about supported releases, see the
+  version-schedule-prompt-link-text: release schedule
```
Comment (Trott) [inline locale/it/index.md:15]: @nodejs/nodejs-it  Can someone provide translations for these phrases (which will be put together to form a sentence in the page)?

File: locale/ja/index.md [hunk 14]
```diff
@@ -11,8 +11,8 @@ labels:
   tagline-lts: 推奨版
   changelog: 変更履歴
   api: API ドキュメント
-  version-schedule-prompt: または、
-  version-schedule-prompt-link-text: LTSのリリーススケジュールをご覧ください
+  version-schedule-prompt: For information about supported releases, see the
+  version-schedule-prompt-link-text: release schedule
```
Comment (Trott) [inline locale/ja/index.md:15]: @nodejs/nodejs-ja  Can someone provide translations for these phrases (which will be put together to form a sentence in the page)?

File: locale/ko/index.md [hunk 15]
```diff
@@ -11,8 +11,8 @@ labels:
   tagline-lts: 안정적, 신뢰도 높음
   changelog: 변경사항
   api: API 문서
-  version-schedule-prompt: LTS 일정은
-  version-schedule-prompt-link-text: 여기서 확인하세요
+  version-schedule-prompt: For information about supported releases, see the
+  version-schedule-prompt-link-text: release schedule
```
Comment (Trott) [inline locale/ko/index.md]: @nodejs/nodejs-ko  Can someone provide translations for these phrases (which will be put together to form a sentence in the page)?

File: locale/pt-br/index.md [hunk 16]
```diff
@@ -11,8 +11,8 @@ labels:
   tagline-lts: Recomendado Para Maioria dos Usuários
   changelog: Changelog
   api: Documentação da API
-  version-schedule-prompt: Ou verifique o
-  version-schedule-prompt-link-text: Cronograma de versões LTS
+  version-schedule-prompt: For information about supported releases, see the
+  version-schedule-prompt-link-text: release schedule
```
Comment (Trott) [inline locale/pt-br/index.md]: @nodejs/nodejs-pt  Can someone provide translations for these phrases (which will be put together to form a sentence in the page)?

File: locale/ro/index.md [hunk 17]
```diff
@@ -11,8 +11,8 @@ labels:
   tagline-lts: Recommended For Most Users
   changelog: Changelog
   api: API Docs
-  version-schedule-prompt: Or have a look at the
-  version-schedule-prompt-link-text: Long Term Support (LTS) schedule
+  version-schedule-prompt: For information about supported releases, see the
+  version-schedule-prompt-link-text: release schedule
```
Comment (Trott) [inline locale/ro/index.md:15]: @nodejs/nodejs-ro  Can someone provide translations for these phrases (which will be put together to form a sentence in the page)?

File: locale/ru/index.md [hunk 18]
```diff
@@ -11,8 +11,8 @@ labels:
   tagline-lts: Рекомендовано для большинства
   changelog: Список изменений
   api: Документация
-  version-schedule-prompt: Или смотрите на
-  version-schedule-prompt-link-text: график LTS
+  version-schedule-prompt: For information about supported releases, see the
+  version-schedule-prompt-link-text: release schedule
```
Comment (Trott) [inline locale/ru/index.md:15]: @nodejs/nodejs-ru  Can someone provide translations for these phrases (which will be put together to form a sentence in the page)?

File: locale/uk/index.md [hunk 19]
```diff
@@ -11,8 +11,8 @@ labels:
   tagline-lts: Рекомендовано для більшості
   changelog: Список змін
   api: Документація
-  version-schedule-prompt: Або подивіться на
-  version-schedule-prompt-link-text: графік LTS
+  version-schedule-prompt: For information about supported releases, see the
+  version-schedule-prompt-link-text: release schedule
```
Comment (Trott) [inline locale/uk/index.md:15]: @nodejs/nodejs-uk  Can someone provide translations for these phrases (which will be put together to form a sentence in the page)?

File: locale/ko/index.md (hunk 15 above)
Comment (taggon) [reply locale/ko/index.md]:
```suggestion
  version-schedule-prompt: 지원되는 릴리스에 관한 정보는
  version-schedule-prompt-link-text: 릴리스 일정을 참고하세요
```
Here it is. :)

File: locale/pt-br/index.md (hunk 16 above)
Comment (TiagoDanin) [inline locale/pt-br/index.md:15]:
```suggestion
  version-schedule-prompt: Para mais informações sobre o suporte das versões, consulte o 
  version-schedule-prompt-link-text: cronograma de lançamentos
```

File: locale/pt-br/index.md [hunk 20]
```diff
@@ -11,8 +11,8 @@ labels:
   tagline-lts: Recomendado Para Maioria dos Usuários
   changelog: Changelog
   api: Documentação da API
-  version-schedule-prompt: Ou verifique o
-  version-schedule-prompt-link-text: Cronograma de versões LTS
+  version-schedule-prompt: For information about supported releases, see the
```
Comment (TiagoDanin) [inline locale/pt-br/index.md]:
```suggestion
  version-schedule-prompt: Para mais informações sobre o suporte das versões, consulte o 
  version-schedule-prompt-link-text: cronograma de lançamentos
```

Comment (ghost) [reply locale/pt-br/index.md]: Remove the duplicated "version-schedule-prompt-link-text" in English so as to let it get passed :)

**Pull request closed without merging**